docs/lib/auth/fragments/ios/signin_next_steps/50_reset_password.md

```diff
@@ -0,0 +1,41 @@
+If you receive `resetPassword`, authentication flow could not proceed without resetting the password. The next step is to invoke `resetPassword` api and follow the reset password flow.
```

Comment:
Hm. I didn't realize that we have both `resetPassword` and `confirmSignInWithNewPassword`. How are they different? Can we add a sentence or two comparing/contrasting them, to each of the two sections?

Reply:
Added a note in both